src/commands/triggerWorkflowRun.ts

```diff
@@ -20,28 +20,17 @@ export function registerTriggerWorkflowRun(context: vscode.ExtensionContext) {
       "github-actions.explorer.triggerRun",
       async (args: TriggerRunCommandOptions | vscode.Uri) => {
         let workflowUri: vscode.Uri | null = null;
+        let gitHubRepoContext: GitHubRepoContext | null = null;
+
         if (args instanceof vscode.Uri) {
           workflowUri = args;
         } else if (args.wf) {
```

Comment:
If we come in with a `vscode.Uri` as `args` won't we end up with `gitHubRepoContext` being `null` and thus always existing this in line 34?